target/social-map-1/CreateAnEvent.html

```diff
@@ -0,0 +1,113 @@
+  </head>
+  <body onload="onload()">
+    <div class="navbar">
+      <p class="navbar-text" onclick="window.location.href = 'map.html'">Social Map</p>
+      <p class="navbar-text" onclick="window.location.href = 'searchUsers.html'">Search Users</p>
+      <p class="navbar-text" onclick="window.location.href = 'nearme.html'">Near Me</p>
+      <div id="user-navbar-section"></div>
+    </div>
+    <div id="events-title">
+      <h1>Create An Event</h1>
+    </div>
+    <div id=event-body>
+      <div id="map-snippet"></div>
+      <div id=event-form class="shaded-box">
+        <form>
+          <label for="event-name">Event Name*: </label>
+          <input type="text" id="event-name" name="event-name" required></textarea>
+          <br><br>
+          
+          <label for="start-date">Start Date*: </label>
+          <input type="date" id="start-date" name="start-date" required>
+          <label for="start-time">Start Time*: </label>
+          <input type="time" id="start-time" name="start-time" required>
+          <br>
+
+          <div id="date-warning" class="warning"></div>
+          <label for="end-date">End Date*: </label>
+          <input type="date" id="end-date" name="end-date" required>
+          <label for="end-time">End Time*: </label>
+          <input type="time" id="end-time" name="end-time" required>
+          <label for="time-zone">Time Zone*: </label>
+          <select id="time-zone" name="time-zone" required>
+            <option value="AST">AST</option>
+            <option value="EST">EST</option>
+            <option value="EDT">EDT</option>
+            <option value="CST">CST</option>
+            <option value="CDT">CDT</option>
+            <option value="MST">MST</option>
+            <option value="MDT">MDT</option>
+            <option value="PST">PST</option>
+            <option value="PDT">PDT</option>
+            <option value="AKST">AKST</option>
+            <option value="AKDT">AKDT</option>
+            <option value="HST">HST</option>
+            <option value="HAST">HAST</option>
+            <option value="HADT">HADT</option>
+            <option value="SST">SST</option>
+            <option value="SDT">SDT</option>
+            <option value="CHST">CHST</option>
+          </select>
+          <br><br>
+
+          <label for="location">Location: </label>
+          <input type="text" id="location" name="location">
+          <input type="text" id="place-id" name="place-id">
+          <br><br>
+
+          <label for="event-details">Event Details: </label>
+          <input type="text" id="event-details" name="event-details">
+          <br><br>
+
+          <label for="privacy">Privacy*: (please choose one)</label>
+          <select id="privacy" onClick="specifiedAttendees(this.value)" name="privacy" required>
+            <option value="public">Public</option>
+            <option value="buddies-only">Buddies</option>
```

Comment:
It looks like this value is specified as "buddies" in some places, but "buddies-only" in others. Intentional?